### ISSUE TITLE
Same headwords, different meanings in gauging "known"

### ISSUE THREAD

**Issue by IvyJL** [Hedera-Lang-Learn/hedera#281]
We created the Hedera lattice to distinguish between different meanings within a headword. So _ab_ is a single headword in Morpheus with two distinct meanings, usually learned at very different times.

ab	by (+ abl.)
ab	from, away from, out of (+abl.)

In our intro book, the "from" meaning is learned in ch. 4, but the "by" meaning not until ch. 14.

When I disambiguate in Hedera, I can choose which of these meanings it actually is in every instance.

But when I compare to a vocab list that only has one of the meanings, the gauge is counting **any** meaning of _ab_ as known.

Here's a tiny tsv vocab list that I imported.
in (+ abl.)	in, on
ab (+ abl.)	from away from
casa, -ae	house

And here's a tiny text
ab casa ambulo
ab milite petor 

In this text the first _ab_ is "from" but the second "by". 

The problem is that when I gauge against that vocab list, it claims I know both uses of _ab_.

**Comment by jtauber** [comment]
When the vocab list item is linked to the lattice, I bet it is being linked to the parent node instead of the narrower sense. We'd need the ability to point the vocab list entry to the right lattice node.

**Comment by IvyJL** [comment]
That sounds right, but I thought that was already being done. Isn't that the main advantage of the lattice over just using whatever the Morpeus headword is?

**Comment by jtauber** [comment]
The issue is there's no automatic way to know how to map the vocab list item to the right node. It would have to be done manually.

**Comment by IvyJL** [comment]
I thought we had talked about looking at both the headword and definition to associate with the right node?
Or asking the uploader to disambiguate, like you would with a text?

**Comment by jtauber** [comment]
The definition isn't necessarily going to be reliable. So yes, we need the uploader to disambiguate, we just haven't done that yet (although it might be captured in an issue somewhere as Patrick and I have definitely talked about it).

**Comment by IvyJL** [comment]
Agreed, the definition isn’t necessarily going to be reliable, but sort of like going with the frequency data as a “best guess” in disambiguating a text, I feel like looking for matching words in the definition will cover a good many of the cases. I mean, below are a few of the things in the most common words.

If a person uploads to their list something like “ab	away from; by”, we could add both of our ab entries to their list.
In fact, perhaps that’s what we should do, add everything with the same sub-lemma and let people delete/downgrade ones they don’t know.

ā	ah!
ā	by (+ abl.)
ā	from, away from,  (+ abl.)

ab	by (+ abl.)
ab	from, away from (+ abl.)

ac	and
ac	than (after comparatives)

atque	and
atque	than (after comparatives)

ut	as; how
ut	that, so that

in	in, on (+ abl.)
in	into, onto (+ acc.)

quam	as … as possible 
quam	in what manner, to what degree, how greatly, how, how much
quam	than (in comparisons)

dē	about, concerning (+ abl.)
dē	down from (+ abl.)

nihil	nothing
nihil	not at all

manus, manūs, f.	band of people
manus, manūs, f.	hand

ubi	when
ubi	where

dum	provided that (+ subj.)
dum	until (+ subj.)
dum	while (+ indic.)

cum	with (+ abl.)
cum	when (+ indic.)
cum	when; since; although (+ subj.)
